### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by lonedfx** [carson-katri/dream-textures#123] [Ideas]
Most excellent plugin! I only have a couple suggestions:

- Reopening the dialog all the time when tweaking things is a bit of a hassle, maybe make the plugin a tab, or allow its dialog to stay open?

- Allow cancelling the current sampling without having to wait for it to finish. With large number of steps and larger textures, this can take a while to complete so if the preview shows this is not going in the right direction, it would be great to be able to stop the sampler.

Awesome work!

**Comment by AlesJelovcan** [comment]
Hi, very nice work on this plugin, love using it!

I am sure just about every other user though of this but anyways, here is my suggestion: automate rendering, so we can do videos without manually clicking for every frame. So, just taking in render result as an init image.

Thank you,  good people behind this plugin!

**Comment by imperioame** [comment]
I'd like to add a suggestion too.
Love the pluggin. I've installed Stable diffusion 'standalone' from another repo (stable-diffusion-ui), so i have it installed twice. Can this plugin be run from another installation directory?
It would probably require a complex implementation and would be prompt to errors

**Comment by NullSenseStudio** [comment]
> * Reopening the dialog all the time when tweaking things is a bit of a hassle, maybe make the plugin a tab, or allow its dialog to stay open?
> * Allow cancelling the current sampling without having to wait for it to finish. With large number of steps and larger textures, this can take a while to complete so if the preview shows this is not going in the right direction, it would be great to be able to stop the sampler.

With the release of [v0.0.6](https://github.com/carson-katri/dream-textures/releases/tag/0.0.6) both of these are included. It's now part of the side panel and you are able to cancel in the middle of generating, though you will have to wait for the model to reload for the next time you generate.

**Comment by 1435thunder** [comment]
It would be a great addition if we could export the individual history /seed to reconstruct /distribute our scene in a blend file.

**Reply by carson-katri** [reply]
History should already persist across different blend files. There’s a metadata format used by InvokeAI that could be included in the future too.

**Reply by 1435thunder** [reply]
i did try it with .6 yesterday will try now. ...
Sorry, i still cannot see any history in the recall prompt window. This is what I meant in my request comment.  the seed is there tho.
Hard to articulate my request /suggestion, but i thought i could select one iteration from the history list and save it individually and or the whole history.
thanks i realy love this add-on

**Reply by carson-katri** [reply]
This is available in 0.0.7, see this guide for more details: https://github.com/carson-katri/dream-textures/blob/main/docs/HISTORY.md#prompt-importexport

**Comment by Kareemsi** [comment]
Hi, can we get a better way of uninstalling and upgrading without having to remove and reinstall Blender? I don't want to have junk files left over from every upgrade and I don't want to have to keep reinstall blender to have a clean update.

**Reply by NullSenseStudio** [reply]
Due to python dependencies being packaged in the addon instead of being installed separately in v0.0.6 it will no longer leave files within Blender once uninstalled.

**Comment by lonedfx** [comment]
Thank you for the update!

**Comment by moebis** [comment]
Is it possible to use the larger sd-v1-4-full-ema.ckpt model?

**Reply by carson-katri** [reply]
It should be possible, but you probably wouldn’t want to use it unless you’re training the model.

**Comment by Xenodimensional** [comment]
Thank you for this amazing plugin! It's a game changer. I am wondering if it is possible (or could be) to batch render variations out and save them automatically ad infinitum?

**Reply by carson-katri** [reply]
Yeah, this is certainly possible.

**Comment by Josiahgil** [comment]
Hi, I'm don't know anything about code, i know it must hard to just add a new feature to stuff, but if possible, I think it would be worth adding a "proxy setting" feature into the dream textures addon installation, sort of like the blender-kit addon, it has a proxy settings.  Thanks for your time, and for this cool addon.

**Reply by carson-katri** [reply]
What does the proxy setting do?

**Reply by Josiahgil** [reply]
Thanks for the quick reply.  Proxy settings allow a computer, or an app to connect to a server using a proxy address, and it provides some security and privacy, also sometimes a proxy can be used to hide internet usage, and allow the use of unlimited high speed internet on a limited plan.

**Reply by carson-katri** [reply]
The model runs offline, so I’m not sure how this would apply to be honest.

**Reply by Josiahgil** [reply]
Oh, that's really cool then. My bad, i had everything mixed up.
